.github/workflows/main.yml

```diff
@@ -0,0 +1,125 @@
+name: main
+
+on:
+  push:
+    branches:
+      - '**'
+    tags:
+      - 'v*'
+  pull_request:
+
+jobs:
+  lint:
+    runs-on: ubuntu-18.04
+
+    steps:
+      - run: |
+          echo "noop"
+
+  python-unit-tests:
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        service: ["extractor", "transformer", "loader", "analyzer"]
+        python-version: [3.7]
```

Comment:
Why not using it directly?

Reply:
fixed (I first tried with python 3.7 and 3.9 but the build fails on 3.9 because the confluent_kafka library seems to miss a dependency...)